service/frontend/nexus_handler.go

```diff
@@ -267,7 +315,39 @@ func (h *nexusHandler) StartOperation(ctx context.Context, operation string, inp
 	}
 	return nil, fmt.Errorf("unhandled response outcome: %T", response.GetOutcome()) //nolint:goerr113
 }
-func (h *nexusHandler) CancelOperation(ctx context.Context, operation, id string, options nexus.CancelOperationOptions) (retErr error) {
+
+func (h *nexusHandler) forwardStartOperation(
```

Comment:
I'd just put a comment here saying that we're using the `Reader` to avoid reading bodies and bypassing serialization.